data/sql/updates/pending_db_world/rev_1646755683709231100.sql

```diff
@@ -0,0 +1,951 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646755683709231100');
+
+DELETE FROM `quest_template_locale` WHERE `locale` = 'zhTW' AND `ID` IN (9794,9795,10455,10456,10457,10486,10487,10488,10489,10502,10503,10504,10505,10506,10510,10511,10512,10516,10517,10518,10524,10525,10526,10542,10543,10544,10545,10555,10556,10557,10560,10565,10566,10567,10580,10581,10584,10594,10607,10608,10609,10614,10615,10617,10618,10620,10632,10657,10671,10674,10675,10682,10690,10709,10710,10711,10712,10713,10714,10715,10717,10718,10720,10721,10722,10723,10724,10742,10747,10748,10749,10753,10770,10771,10783,10784,10785,10786,10787,10795,10796,10797,10798,10802,10803,10805,10806,10810,10812,10819,10821,10825,10829,10830,10843,10845,10846,10851,10853,10859,10860,10893,10894,10910,10927,10928,10974,10976,10983,10989,10995,10996,10997,10998,11000,11004,11005,11006,11008,11009,11010,11021,11023,11024,11025,11028,11029,11030,11036,11040,11043,11047,11051,11056,11057,11058,11059,11061,11062,11065,11066,11072,11073,11078,11079,11080,11085,11091,11093,11098,11119,11337,11341,11368,11370,11371,11372,11373,11375,11376,11514,11622,11683,11885,11893,11916,12018,12021,12024,12025,12087,12108,12156,12234,12258,12260,12269,12271,12275,12276,12277,12434,12446,12490,12502,12509,12517,12518,12519,12541,12547,12852,12911,14179,14182);
+INSERT INTO `quest_template_locale` VALUES
+(12547,"zhTW","啟動符文","這個巨大的存在似乎停止了運作。你注意到它胸口有一個符文狀的空槽。$B$B也許你可以在附近找到這個失蹤的符文。","取得塑造者殿堂內的終末符文，然後帶去找伊提米迪安。","","到安戈洛環形山的拉卡利油沼找伊提米迪安。","","","","",0),
```

Comment:
Please use quotation marks ' instead of doubled ones " around every strings to keep a line.

Reply:
Thank you, I have made that change.